integration_tests/Project.toml

```
@@ -74,6 +74,7 @@ StaticArrays = "1.2"
 StatsBase = "0.33"
 StochasticDiffEq = "6.41"
 SurfaceFluxes = "0.4"
+SOCRATESSingleColumnForcings = "0.6.0"
```

```suggestion
SOCRATESSingleColumnForcings = "0.6"
```

I've updated the dependencies in #1333, so we can discard the manifest file changes.